lib/smart_answer_flows/benefit-cap-calculator.rb

```diff
@@ -127,7 +127,7 @@ def define
 
         next_node do |response|
           if response == "none"
-            outcome :outcome_not_affected_future
+            question :housing_benefit_amount?
```

Comment:
While I understand the changes to `benefit-cap-calculator-responses-and-expected-results.yml`, I'm not 100% sure I understand the changes to the regression test artefacts. Ideally I think we'd try to improve the explanation in the commit note.